### ISSUE TITLE
ERROR: unable to download video data: HTTP Error 403: Forbidden

### ISSUE THREAD

**Issue by piroXpower** [yt-dlp/yt-dlp#8423]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm asking a question and **not** reporting a bug or requesting a feature
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I'm running yt-dlp version **2023.10.13** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Please make sure the question is worded well enough to be understood

I RUN THE COMMAND IN MY UBUNTU 20 

yt-dlp -v https://www.youtube.com/watch?v=f6wtF_2eyrU

And i got some errors 

o%2Fwebm&gir=yes&clen=51051295&dur=3703.581&lmt=16636107302854
91&mt=1698134235&fvip=4&keepalive=yes&fexp=24007246&beids=2435
0018&c=ANDROID&txp=6211224&sparams=expire%2Cei%2Cip%2Cid%2Cita
g%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Ccl
en%2Cdur%2Clmt&sig=AGM4YrMwRQIhAPlhykGqKCULLuhVfdaTKQb7WwsvBI2
Rw7-FPS_rL49kAiBhhlrFWiAzOjBEFc4DkcylstvQNuaM3aooisVe6AjU5Q%3D
%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpcm2cms%2Cpl%2Cini
tcwndbps&lsig=AK1ks_kwRgIhAJ7DHWnwx7jyDUXnROfemjnKyjGPjo4Ek7qC
T_yUBSZmAiEAwljNQtnrbp5RJ2sVM3odRIMlPJQlX2QcOEnztAciXKs%3D"
ERROR: unable to download video data: HTTP Error 403: Forbidde
n
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeD
L.py", line 4051, in urlopen
    return self._request_director.send(req)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/networki
ng/common.py", line 114, in send
    response = handler.send(request)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/networki
ng/_helper.py", line 204, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/networki
ng/common.py", line 325, in send
    return self._send(request)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/networki
ng/_requests.py", line 340, in _send
root@None:~/YukkiMusicBot# yt-dlp -v http://www.youtube.com/wa
tch?v=NGLxoKOvzu4
[debug] Command-line config: ['-v', 'http://www.youtube.com/wa
tch?v=NGLxoKOvzu4']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf
-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.10.13 [b634ba742]
[debug] Lazy loading extractors is disabled
[debug] Python 3.8.10 (CPython x86_64 64bit) - Linux-5.4.0-42-
generic-x86_64-with-glibc2.29 (OpenSSL 1.1.1f  31 Mar 2020, gl
ibc 2.31)
[debug] exe versions: ffmpeg 4.2.7, ffprobe 4.2.7
[debug] Optional libraries: Cryptodome-3.19.0, brotli-1.1.0, c
ertifi-2019.11.28, mutagen-1.47.0, requests-2.31.0, secretstor
age-2.3.1, sqlite3-3.31.1, urllib3-2.0.7, websockets-12.0
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests
[debug] Loaded 1890 extractors
[youtube] Extracting URL: http://www.youtube.com/watch?v=NGLxo
KOvzu4
[youtube] NGLxoKOvzu4: Downloading webpage
[youtube] NGLxoKOvzu4: Downloading ios player API JSON
[youtube] NGLxoKOvzu4: Downloading android player API JSON
[youtube] NGLxoKOvzu4: Downloading m3u8 information
[debug] Sort order given by extractor: quality, res, fps, hdr:
12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps,
 hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang,
proto, size, br, asr, vext, aext, hasaud, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] NGLxoKOvzu4: Downloading 1 format(s): 616+251
[debug] Invoking hlsnative downloader on "https://manifest.goo
glevideo.com/api/manifest/hls_playlist/expire/1698156625/ei/8X
s3Za39DfyN3LUPhdKA8Ak/ip/103.211.56.132/id/3462f1a0a3afceee/it
ag/616/source/youtube/requiressl/yes/ratebypass/yes/pfa/1/wft/
1/sgovp/clen%3D144337300%3Bdur%3D235.026%3Bgir%3Dyes%3Bitag%3D
356%3Blmt%3D1665698742712285/hls_chunk_host/rr2---sn-2gxjpv0no
xujvh-qxae.googlevideo.com/mh/YD/mm/31,29/mn/sn-2gxjpv0noxujvh
-qxae,sn-qxaeense/ms/au,rdu/mv/m/mvi/2/pl/24/initcwndbps/93750
0/vprv/1/playlist_type/DVR/dover/13/txp/5532434/mt/1698134713/
fvip/3/short_key/1/keepalive/yes/fexp/24007246/beids/24350018/
sparams/expire,ei,ip,id,itag,source,requiressl,ratebypass,pfa,
wft,sgovp,vprv,playlist_type/sig/AGM4YrMwRQIgblPE1Eh8Q1wCLykAt
SmjU4qAF9_qQFqghN2yvDmru-8CIQCNrxfz-fIfNma8LlhO5imLRJgPfI_-xLN
nGaarHfffqQ%3D%3D/lsparams/hls_chunk_host,mh,mm,mn,ms,mv,mvi,p
l,initcwndbps/lsig/AK1ks_kwRQIhAPHJAYSRRDUIhQv5ddQLFi07RCELO1G
53WK7Liid4k6CAiA2wRlAnrGQ9nJOBZR74xKo34w15JXBGHHypAhbtcG1sQ%3D
%3D/playlist/index.m3u8"
[hlsnative] Downloading m3u8 manifest
[hlsnative] Total fragments: 46
[download] Destination: Jason Derulo - Swalla (feat. Nicki Min
aj & Ty Dolla $ign) [Official Music Video] [NGLxoKOvzu4].f616.
mp4
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 1 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 1 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 1 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 1 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 1 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 1 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 1 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 1 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 1 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 1 (10/10)...
[download] fragment not found; Skipping fragment 1 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 2 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 2 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 2 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 2 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 2 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 2 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 2 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 2 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 2 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 2 (10/10)...
[download] fragment not found; Skipping fragment 2 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 3 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 3 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 3 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 3 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 3 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 3 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 3 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 3 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 3 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 3 (10/10)...
[download] fragment not found; Skipping fragment 3 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 4 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 4 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 4 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 4 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 4 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 4 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 4 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 4 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 4 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 4 (10/10)...
[download] fragment not found; Skipping fragment 4 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 5 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 5 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 5 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 5 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 5 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 5 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 5 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 5 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 5 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 5 (10/10)...
[download] fragment not found; Skipping fragment 5 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 6 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 6 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 6 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 6 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 6 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 6 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 6 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 6 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 6 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 6 (10/10)...
[download] fragment not found; Skipping fragment 6 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 7 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 7 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 7 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 7 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 7 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 7 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 7 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 7 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 7 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 7 (10/10)...
[download] fragment not found; Skipping fragment 7 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 8 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 8 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 8 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 8 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 8 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 8 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 8 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 8 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 8 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 8 (10/10)...
[download] fragment not found; Skipping fragment 8 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 9 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 9 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 9 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 9 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 9 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 9 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 9 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 9 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 9 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 9 (10/10)...
[download] fragment not found; Skipping fragment 9 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 10 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 10 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 10 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 10 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 10 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 10 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 10 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 10 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 10 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 10 (10/10)...
[download] fragment not found; Skipping fragment 10 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 11 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 11 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 11 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 11 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 11 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 11 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 11 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 11 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 11 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 11 (10/10)...
[download] fragment not found; Skipping fragment 11 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 12 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 12 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 12 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 12 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 12 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 12 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 12 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 12 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 12 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 12 (10/10)...
[download] fragment not found; Skipping fragment 12 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 13 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 13 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 13 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 13 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 13 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 13 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 13 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 13 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 13 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 13 (10/10)...
[download] fragment not found; Skipping fragment 13 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 14 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 14 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 14 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 14 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 14 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 14 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 14 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 14 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 14 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 14 (10/10)...
[download] fragment not found; Skipping fragment 14 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 15 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 15 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 15 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 15 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 15 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 15 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 15 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 15 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 15 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 15 (10/10)...
[download] fragment not found; Skipping fragment 15 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 16 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 16 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 16 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 16 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 16 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 16 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 16 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 16 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 16 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 16 (10/10)...
[download] fragment not found; Skipping fragment 16 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 17 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 17 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 17 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 17 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 17 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 17 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 17 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 17 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 17 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 17 (10/10)...
[download] fragment not found; Skipping fragment 17 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 18 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 18 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 18 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 18 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 18 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 18 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 18 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 18 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 18 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 18 (10/10)...
[download] fragment not found; Skipping fragment 18 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 19 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 19 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 19 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 19 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 19 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 19 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 19 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 19 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 19 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 19 (10/10)...
[download] fragment not found; Skipping fragment 19 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 20 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 20 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 20 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 20 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 20 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 20 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 20 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 20 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 20 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 20 (10/10)...
[download] fragment not found; Skipping fragment 20 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 21 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 21 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 21 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 21 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 21 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 21 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 21 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 21 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 21 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 21 (10/10)...
[download] fragment not found; Skipping fragment 21 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 22 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 22 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 22 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 22 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 22 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 22 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 22 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 22 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 22 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 22 (10/10)...
[download] fragment not found; Skipping fragment 22 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 23 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 23 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 23 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 23 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 23 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 23 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 23 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 23 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 23 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 23 (10/10)...
[download] fragment not found; Skipping fragment 23 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 24 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 24 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 24 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 24 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 24 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 24 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 24 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 25 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 25 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 25 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 25 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 25 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 25 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 25 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 25 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 25 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 25 (10/10)...
[download] fragment not found; Skipping fragment 25 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 26 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 26 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 26 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 26 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 26 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 26 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 26 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 26 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 26 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 26 (10/10)...
[download] fragment not found; Skipping fragment 26 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 27 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 27 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 27 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 27 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 27 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 27 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 27 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 27 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 27 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 27 (10/10)...
[download] fragment not found; Skipping fragment 27 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 28 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 28 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 28 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 28 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 28 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 28 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 28 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 28 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 28 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 28 (10/10)...
[download] fragment not found; Skipping fragment 28 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 29 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 29 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 29 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 29 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 29 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 29 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 29 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 29 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 29 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 29 (10/10)...
[download] fragment not found; Skipping fragment 29 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 30 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 30 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 30 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 30 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 30 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 30 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 30 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 30 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 30 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 30 (10/10)...
[download] fragment not found; Skipping fragment 30 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 31 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 31 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 31 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 31 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 31 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 31 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 31 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 31 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 31 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 31 (10/10)...
[download] fragment not found; Skipping fragment 31 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 32 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 32 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 32 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 32 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 32 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 32 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 32 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 32 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 32 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 32 (10/10)...
[download] fragment not found; Skipping fragment 32 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 33 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying frag
ment 33 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 29 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 29 (10/10)...
[download] fragment not found; Skipping fragment 29 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 30 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 30 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 30 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 30 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 30 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 30 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 30 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 30 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 30 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 30 (10/10)...
[download] fragment not found; Skipping fragment 30 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 31 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 31 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 31 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 31 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 31 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 31 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 31 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 31 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 31 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 31 (10/10)...
[download] fragment not found; Skipping fragment 31 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 32 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 32 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 32 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 32 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 32 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 32 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 32 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 32 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 32 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 32 (10/10)...
[download] fragment not found; Skipping fragment 32 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 33 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 33 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 33 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 33 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 33 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 33 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 33 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 33 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 33 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 33 (10/10)...
[download] fragment not found; Skipping fragment 33 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 34 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 34 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 34 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 34 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 34 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 34 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 34 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 34 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 34 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 34 (10/10)...
[download] fragment not found; Skipping fragment 34 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 35 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 35 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 35 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 35 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 35 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 35 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 35 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 35 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 35 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 35 (10/10)...
[download] fragment not found; Skipping fragment 35 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 36 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 36 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 36 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 36 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 36 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 36 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 36 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 36 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 36 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 36 (10/10)...
[download] fragment not found; Skipping fragment 36 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 37 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 37 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 37 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 37 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 37 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 37 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 37 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 37 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 37 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 37 (10/10)...
[download] fragment not found; Skipping fragment 37 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 38 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 38 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 38 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 38 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 38 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 38 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 38 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 38 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 38 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 38 (10/10)...
[download] fragment not found; Skipping fragment 38 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 39 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 39 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 39 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 39 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 39 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 39 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 39 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 39 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 39 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 39 (10/10)...
[download] fragment not found; Skipping fragment 39 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 40 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 40 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 40 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 40 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 40 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 40 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 40 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 40 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 40 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 40 (10/10)...
[download] fragment not found; Skipping fragment 40 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 41 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 41 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 41 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 41 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 41 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 41 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 41 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 41 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 41 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 41 (10/10)...
[download] fragment not found; Skipping fragment 41 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 42 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 42 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 42 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 42 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 42 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 42 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 42 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 42 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 42 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 42 (10/10)...
[download] fragment not found; Skipping fragment 42 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 43 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 43 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 43 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 43 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 43 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 43 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 43 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 43 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 43 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 43 (10/10)...
[download] fragment not found; Skipping fragment 43 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 44 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 44 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 44 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 44 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 44 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 44 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 44 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 44 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 44 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 44 (10/10)...
[download] fragment not found; Skipping fragment 44 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 45 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 45 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 45 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 45 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 45 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 45 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 45 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 45 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 45 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 45 (10/10)...
[download] fragment not found; Skipping fragment 45 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 46 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 46 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 46 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 46 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 46 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 46 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 46 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 46 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 46 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 46 (10/10)...
[download] fragment not found; Skipping fragment 46 ...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 47 (1/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 47 (2/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 47 (3/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 47 (4/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 47 (5/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 47 (6/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 47 (7/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 47 (8/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 47 (9/10)...
[download] Got error: HTTP Error 403: Forbidden. Retrying fragment 47 (10/10)...
[download] fragment not found; Skipping fragment 47 ...
ERROR: The downloaded file is empty
  File "/usr/local/bin/yt-dlp", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/__init__.py", line 1008, in main
    _exit(*variadic(_real_main(argv)))
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/__init__.py", line 998, in _real_main
    return ydl.download(all_urls)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 3510, in download
    self.__download_wrapper(self.extract_info)(
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 3485, in wrapper
    res = func(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 1556, in extract_info
    return self.__extract_info(url, self.get_info_extractor(key), download, extra_info, process)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 1567, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 1723, in __extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 1782, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 2921, in process_video_result
    self.process_info(new_info)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 3369, in process_info
    partial_success, real_download = self.dl(fname, new_info)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 3108, in dl
    return fd.download(name, new_info, subtitle)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/downloader/common.py", line 455, in download
    ret = self.real_download(filename, info_dict)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/downloader/hls.py", line 375, in real_download
    return self.download_and_append_fragments(ctx, fragments, info_dict)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/downloader/fragment.py", line 527, in download_and_append_fragments
    return self._finish_frag_download(ctx, info_dict)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/downloader/fragment.py", line 301, in _finish_frag_download
    self.report_error('The downloaded file is empty')
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 1045, in report_error
    self.trouble(f'{self._format_err("ERROR:", self.Styles.ERROR)} {message}', *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 973, in trouble
    tb_data = traceback.format_list(traceback.extract_stack())
[debug] Invoking http downloader on "https://rr2---sn-2gxjpv0noxujvh-qxae.googlevideo.com/videoplayback?expire=1698156625&ei=8Xs3ZdftGJuy4t4PiaKx-Ak&ip=103.211.56.132&id=o-APWB4hDPo4lmMO3YmqyKv_atw7wzMUmUxY1OMmis62jb&itag=251&source=youtube&requiressl=yes&mh=YD&mm=31%2C29&mn=sn-2gxjpv0noxujvh-qxae%2Csn-qxaeense&ms=au%2Crdu&mv=m&mvi=2&pl=24&initcwndbps=937500&spc=UWF9f0ncDdRTHBtOt6-kDpayysfVbRQ&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=3732340&dur=235.041&lmt=1570330634919890&mt=1698134713&fvip=3&keepalive=yes&fexp=24007246&beids=24350018&c=ANDROID&txp=5531432&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AGM4YrMwRgIhALxNYbteFjxWGMHR_uTE9EAu7rbIKvJGNRWcEw3Mbam3AiEA1bogjHEOOv46FrF2irfaBgxshxN6WfoVLMKCxxiNqZc%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AK1ks_kwRQIgSGPJ744EBMDCOXtKY39KUY-h6X4QKd4pAqoUSVP1rWsCIQCNn5-50Q4a7hit8l3enPW1iXFDVa2d4o8rWlhaOmqHHA%3D%3D"
ERROR: unable to download video data: HTTP Error 403: Forbidden
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 4051, in urlopen
    return self._request_director.send(req)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/networking/common.py", line 114, in send
    response = handler.send(request)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/networking/_helper.py", line 204, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/networking/common.py", line 325, in send
    return self._send(request)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/networking/_requests.py", line 340, in _send
    raise HTTPError(res, redirect_loop=max_redirects_exceeded)
yt_dlp.networking.exceptions.HTTPError: HTTP Error 403: Forbidden
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 3369, in process_info
    partial_success, real_download = self.dl(fname, new_info)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 3108, in dl
    return fd.download(name, new_info, subtitle)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/downloader/common.py", line 455, in download
    ret = self.real_download(filename, info_dict)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/downloader/http.py", line 364, in real_download
    establish_connection()
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/downloader/http.py", line 120, in establish_connection
    ctx.data = self.ydl.urlopen(request)
  File "/usr/local/lib/python3.8/dist-packages/yt_dlp/YoutubeDL.py", line 4073, in urlopen
    raise _CompatHTTPError(e) from e
yt_dlp.networking.exceptions._CompatHTTPError: HTTP Error 403: Forbidden

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

**Comment by cuevana8** [comment]
+1..[.](https://github.com/flslib/fl-studio-cracked)[@@@](https://github.com/yt-dlp/yt-dlp/issues/8423#issue-1958741509)

**Comment by piroXpower** [comment]
download machine is from same country

**Comment by piroXpower** [comment]
> did you use the download machine from another country ? should be in the same country

No from same country I did

**Comment by bashonly** [comment]
Duplicate of #7860

Your IP has been blocked by Youtube

**Comment by SantiiRepair** [comment]
Same here but when downloading any 9gag video
